src/components/pages/maintenance/ruleDetails/ruleDetails.js

```diff
@@ -172,27 +176,27 @@ export class RuleDetails extends Component {
           <div className="rule-stat-container">
             <div className="rule-stat-cell">
               <div className="rule-stat-header">Total</div>
-              <div className="rule-stat-value">{renderCount(counts.total)}</div>
+              <div className="rule-stat-value">{renderUndefined(counts.total)}</div>
             </div>
             <div className="rule-stat-cell">
               <div className="rule-stat-header">Open</div>
-              <div className="rule-stat-value">{renderCount(counts.open)}</div>
+              <div className="rule-stat-value">{renderUndefined(counts.open)}</div>
             </div>
             <div className="rule-stat-cell">
               <div className="rule-stat-header">Ack</div>
-              <div className="rule-stat-value">{renderCount(counts.acknowledged)}</div>
+              <div className="rule-stat-value">{renderUndefined(counts.acknowledged)}</div>
             </div>
             <div className="rule-stat-cell">
               <div className="rule-stat-header">Closed</div>
-              <div className="rule-stat-value">{renderCount(counts.closed)}</div>
+              <div className="rule-stat-value">{renderUndefined(counts.closed)}</div>
             </div>
             <div className="rule-stat-cell">
               <div className="rule-stat-header">Last Event</div>
               <div className="rule-stat-value">
                 {
                   selectedAlert.lastOccurrence
-                    ?  <LastTriggerRenderer context={({ t: this.props.t })} value={selectedAlert.lastOccurrence} />
-                    : '--'
+                    ? <LastTriggerRenderer context={({ t: this.props.t })} value={selectedAlert.lastOccurrence} />
```

Comment:
Should LastTriggerRenderer handle the undefined ("--") case?  Also, SeverityRenderer below.

Reply:
Nah, the grid has it's own empty value handler in the `pcsGridConfig` file.

Reply:
OK.